create_repository.sh

```diff
@@ -19,7 +19,7 @@ apt-ftparchive sources . \
     | tee "$where"/Sources \
     | gzip -9 > "$where"/Sources.gz
 
-apt-ftparchive packages "$where" \
+apt-ftparchive -o 'Packages::Extenstions=.deb,.ddeb' packages "$where" \
```

Comment:
Did you test this? `Extenstions` doesn't sound correct.

Reply:
Yes, I've tested it. The option is specified in the [apt-ftparchive manpage](https://manpages.debian.org/jessie/apt-utils/apt-ftparchive.1.en.html).

Reply:
But there it's `Packages::Extensions`, not `Packages::Extenstions`...

Reply:
Oops, I missed that. 🤦

However, turns out the option isn't required for it to include .ddeb packages (That's why it works despite getting the option name wrong). I'll rebase while removing the create_repository.sh part and re-submit it.